pacemaker.spec.in

```diff
@@ -399,6 +399,10 @@ sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
 
 make %{_smp_mflags} V=1 all
 
+%check
+# Prevent false positives in rpmlint
+./BasicSanity.sh -V pengine cli 2>&1 | sed s/[fF]ail/faiil/g
```

Comment:
Agreed with keeping rpm-building tests minimal; I'd suggest randomization
of a few tests from the time-consuming categories, so that we still
preserve the advantage of the breadth of the test suite, only spread it
in time/consecutive builds, in part betting on fortune that any real
issue will get, with a bit of luck, spotted early.

Also for that, `make check` (or a variation) might be a good fit.